### PR TITLE
MVAD edit pass

### PR DESCRIPTION
**Pull request by mikekistler**
This PR just updates descriptions to correct grammar and style.

The original changes were made to the openapi.json by our vendor and I reflected those changes back into the TypeSpec to keep the two in sync.

### CODE REVIEW

**Comment by mikekistler** [comment]
SDKs are still in Beta so change to readOnly is allowed.

File: specification/cognitiveservices/AnomalyDetector/univariate/models.tsp [hunk 1]
```diff
@@ -44,150 +44,149 @@ enum TimeGranularity {
...
 """)
   expectedValues: float32[];
 
   @doc("""
-UpperMargins contain upper margin of each input point. UpperMargin is used to
-calculate upperBoundary, which equals to expectedValue + (100 -
-marginScale)*upperMargin. Anomalies in response can be filtered by
-upperBoundary and lowerBoundary. By adjusting marginScale value, less
-significant anomalies can be filtered in client side. The index of the array is
+UpperMargins contains the upper margin of each input point. UpperMargin is used to
```
Comment (heaths) [inline specification/cognitiveservices/AnomalyDetector/univariate/models.tsp:124]: Nit: almost no other docs in this PR reference the property name in the docs themselves. For this and LowerMargins, couldn't the doc start "The (upper|lower) margin ..."?

Comment (heaths) [reply specification/cognitiveservices/AnomalyDetector/univariate/models.tsp:124]: I see a couple others as well for this tsp, but not some others.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
The LintDiff and ModelVAlidation errors weren't introduced here.

**Review by heaths — APPROVED** [review]

**Pull request merged**